CHANGELOG.md

```diff
@@ -14,6 +14,17 @@ and includes an additional section for migration notes.
 - *Security* - Vulnerabilities fixes and changes.
 
 ## [Unreleased]
+### Changed
+- *ORCA-537*
+  - Renamed step-function `OrcaCopyToGlacierWorkflow` to `OrcaCopyToArchiveWorkflow`.
+  - Renamed lambda `copy_to_glacier` to `copy_to_archive`, or `PREFIX_copy_to_orca` if looking at the name in AWS.
+    Output of lambda and Terraform updated to match. See Migration Notes below.
+
+### Migration Notes
+- If utilizing the `copied_to_glacier` [output property](https://github.com/nasa/cumulus-orca/blob/15e5868f2d1eead88fb5cc8f2e055a18ba0f1264/tasks/copy_to_glacier/schemas/output.json#L47) of `copy_to_glacier`, 
+  rename to new key `copied_to_archive`.
```

Comment:
Thinking this should be `copied_to_orca` as this is forward facing to the user and will better define what was sent to ORCA.

Reply:
If they're receiving output from an Orca lambda, they should already know the context. My understanding was the we were only worried about what they say when looking through AWS resources.